src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -70,8 +71,22 @@ type State = {|
     message: string,
 |};
 
+function convertAssigneesToSelectorItems(approvers: Array<TaskCollabAssignee>): SelectorItems {
+    return approvers.map(({ target }) => {
+        const newSelectorItem: SelectorItem = {
```

Comment:
Consider using an implicit return, since the variable is returned immediately.

Reply:
we may drop this function once we determine if it's safe to use `Array<TaskCollabAssignee>` throughout the feature. If we find a continued need, we can factor this more tersely